react/components/Tabs/Tab.js

```diff
@@ -28,7 +28,9 @@ class Tab extends Component {
           fullWidth ? 'w-100' : ''
         } ${tabStyle}
         v-mid relative h-regular ph6 t-body bg-transparent outline-0
-        `}>
+        ${className}`}
+        style={{ minWidth: 128 }} // review here
```

Comment:
hardcoded ?

Reply:
It will be removed and changed to a class.

Reply:
Updated.